deepchem/feat/molecule_featurizers/rdkit_descriptors.py

```diff
@@ -96,7 +96,9 @@ def __init__(self,
         self.reqd_properties = {}
         self.normalized_desc: Dict[str, Callable] = {}
 
-        all_descriptors = {name: func for name, func in Descriptors.descList}
+        self.descriptors = all_descriptors = {
```

Comment:
Nit, but this is a slightly awkward construction. Let's just do
```
all_descriptors = {...}
self.descriptors = all_descriptors
```
More verbose but less confusing

Reply:
Thanks for your feedback! I changed the syntax to what you suggested. It now reads:
```python
all_descriptors = {name: func for name, func in Descriptors.descList}
self.descriptors = all_descriptors
```